### PR TITLE
Production: Add SMD soldering and testing chapters

### LINKED ISSUE #48: Production
Integration of content from previous OER (cos-h)

Finishing of chapters
- SMD soldering
- THT soldering
- Testing

### PR DESCRIPTION
**Pull request by martinjaeger**
Rendered page can be seen [here](https://learn.libre.solar/branch/production-new/production/).

This PR will make the Production chapter complete.

### CODE REVIEW

File: docs/production/smd.md [hunk 1]
```diff
@@ -1,7 +1,70 @@
...
+<figcaption><b>Video 2.</b> Manual SMD parts placement.</figcaption>
+</center>
+
+Once the step is completed, an inspection should be done to see if the pins are seated correctly based on the markings on the PCB.
+
 ## Reflow oven
+
+A **reflow oven** is suitable for fast soldering of entire boards in one go. If not available, also a heat gun can be used to partially heat the solder.
+
+Fig. 3 shows the reflow oven available in the [FabLab Fabulous St. Pauli](http://www.fablab-hamburg.org/). It is a very common type used for small scale SMD assembly.
```
Comment (Gretel5X) [inline docs/production/smd.md:57]: The link is rendered with a weird space before the dot, maybe have a look there

File: docs/production/pcbs.md [hunk 2]
```diff
@@ -14,7 +20,7 @@ However, the Gerber export is a bit more complicated than providing native files
 
 After the board is opened in Pcbnew, the solder mask clearance and min width must be set to zero. This is important because the manufacturers normally want to define the clearance themselves based on their process. Setting it to zero makes it easier for them to shrink/enlarge the pads for the solder mask. The dialogue can be found under *File > Board Setup* as shown in Figure 1.
 
-<fig-caption src="production/pcbs_kicad_pads_clearance.png" caption="Pads and Mask Clearance dialog (KiCad v5.1.4)" num="1" />
+<fig-caption src="production/pcbs_kicad_pads_clearance.png" caption="Pads and Mask Clearance dialog (KiCad v5.1.4)" num="2" />
```
Comment (Gretel5X) [inline docs/production/pcbs.md]: clearance

**Review by Gretel5X — APPROVED** [review]
Very nice!

File: docs/production/smd.md (hunk 1 above)
Comment (martinjaeger) [reply docs/production/smd.md:57]: Mhh, not sure why there is this space, I guess it's from the Vue component that renders external links and adds the symbol at the end... so we probably can't do anything about it.

File: docs/production/pcbs.md (hunk 2 above)
Comment (martinjaeger) [reply docs/production/pcbs.md]: What do you mean by clearance? (I see that the figure number is wrong in the text, though. will fix that)

File: docs/production/smd.md (hunk 1 above)
Comment (Gretel5X) [reply docs/production/smd.md:57]: No biggy :)

File: docs/production/pcbs.md (hunk 2 above)
Comment (Gretel5X) [reply docs/production/pcbs.md]: In the caption its written with capital "C", is that intended?

Comment (martinjaeger) [reply docs/production/pcbs.md]: Ah, right. No, not really. Will fix it and also write mask with lower-case letter.

**Pull request merged**